proof-of-file-existence/pofe/.github/workflows/build.yml

```diff
@@ -0,0 +1,59 @@
+# This is a basic workflow that is manually triggered
+
+name: pofe
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on: [push, pull_request]
+
+# This workflow makes x86_64 binaries for mac, windows, and linux.
+
+
+jobs:
+  mac-windows:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        targetos: [windows, darwin]
+    name: pofe for ${{ matrix.targetos }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
+        env:
+          GOOS: ${{ matrix.targetos }}
+
+      - name: Compile
+        run: make
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: pofe ${{ matrix.targetos }}
+          path: build
```

Comment:
is there a need to upload the build?

Reply:
Nope - came standard with starport scaffolding, but I'll remove it

Reply:
oh okay, then you can leave it. I didnt know it came with it